lib/crds/redirects.rb

```diff
@@ -0,0 +1,58 @@
+require_relative '../utils/colorized_string'
+require 'csv'
+
```

Comment:
Thinking as a developer who would be new to this project and not be familiar Ruby, I'm thinking of the ways that cognitive overhead can be decreased. This file is called `redirects.rb`, but it doesn't contain the actual redirects. Instead it is responsible for writing the redirects. So would it be better names `redirects_writer` or something like that?